pilot/pkg/proxy/envoy/v2/discovery.go

```diff
@@ -371,7 +371,7 @@ func (s *DiscoveryServer) clearCache() {
 // ConfigUpdate implements ConfigUpdater interface, used to request pushes.
 // It replaces the 'clear cache' from v1.
 func (s *DiscoveryServer) ConfigUpdate(full bool) {
-	inboundConfigUpdates.Add(1)
+	incrementWith(configUpdatesCtx, inboundUpdates)
```

Comment:
This is definitely more difficult to follow than before, but if that is the opencensus API then we dont have a choice.

Reply:
Yes. I've struggled with the balance of trying to establish a layer on top of raw OC that hides some of the verbosity without going too far towards mimicking the prometheus ux.

We could imagine adding something like:

```golang
type Counter interface {
   func Add(value int64)
}

type withContext struct {
    Counter

    measure stats.Int64Measure
    tags context.Context
}

var inboundUpdates = stats.Int64("pilot_inbound_updates", "Total number of updates received by pilot.", stats.UnitDimensionless)

var inboundConfigUpdates = &withContext{
   measure: inboundUpdates, 
   tags: tag.New(tag.Upsert(typeTag, "config")),
}
```